guides/common/modules/proc_configuring-remote-execution-for-pull-client-smart-proxy.adoc

```diff
@@ -1,10 +1,11 @@
 [id="configuring-remote-execution-for-pull-client_{context}"]
-= Configuring remote execution for pull client
```

Comment:
Isn't this change the same as the above? Is the test redundant and can be re-structured so that its not copied?

Reply:
I'll try to merge the two modules into one. It's in two places - one for Foreman server and one for Smart Proxy.